var/spack/repos/builtin/packages/mpich/package.py

```diff
@@ -81,6 +81,15 @@ class Mpich(AutotoolsPackage):
             description='Enable Argobots support')
     variant('fortran', default=True, description='Enable Fortran support')
 
+    variant(
+        'two-level-namespace',
```

Comment:
```suggestion
        'two_level_namespace',
```

`-` in variants are confusing

Reply:
Fixed